easybuild/easyblocks/p/python.py

```diff
@@ -137,6 +149,34 @@ def configure_step(self):
 
         super(EB_Python, self).configure_step()
 
+    def build_step(self, *args, **kwargs):
+        """Custom build procedure for Python, ensure stack size limit is set to 'unlimited' (if desired)."""
+
+        if self.cfg['ulimit_unlimited']:
+            # determine current stack size limit
+            (out, _) = run_cmd("ulimit -s")
+            curr_ulimit_s = out.strip()
+
+            # figure out hard limit for stack size limit;
+            # this determines whether or not we can use "ulimit -s unlimited"
+            (out, _) = run_cmd("ulimit -s -H")
+            max_ulimit_s = out.strip()
+
+            if curr_ulimit_s == UNLIMITED:
+                self.log.info("Current stack size limit is %s: OK", curr_ulimit_s)
+            elif max_ulimit_s == UNLIMITED:
+                self.log.info("Current stack size limit is %s, setting it to %s for build...",
+                              UNLIMITED, curr_ulimit_s)
```

Comment:
The order of the two variables should be inverted:
`
self.log.info("Current stack size limit is %s, setting it to %s for build...",
                    curr_ulimit_s, UNLIMITED)
`

Reply:
Nice catch, fixed now...